doc/api/esm.md

```diff
@@ -2,6 +2,22 @@
 
 <!--introduced_in=v8.5.0-->
 <!-- type=misc -->
+<!-- YAML
+added: v8.5.0
+changes:
+  - version: v12.0.0
+    pr-url: https://github.com/nodejs/node/pull/26745
+    description: Add support for `.js` files via `package.json` `"type"` field.
```

Comment:
Technically `.js` files were always supported, just not as ES modules until 12.

Reply:
This page is about ESM, and this lists the changes of a section named `# Modules: ECMAScript modules`. Do you think the context is enough to disambiguate what I mean?

FYI here's how it looks like on the HTML version:
![image](https://user-images.githubusercontent.com/14309773/94441005-86a19c00-01a2-11eb-939f-22393ebed72a.png)

Reply:
Before 12, `.js` files were supported but always as CommonJS; like `.cjs` files now.

You could just say “Add support for `.js` files as ES modules via `package.json` `"type"` field.”

Reply:
```suggestion
    description: Add support for ES modules using `.js` file extension via `package.json` `"type"` field.
```